code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -288,8 +279,8 @@
 	set_item_state(itemstring)
 
 /obj/item/part/gun/frame/kalash
-	name = "Homemade Kalash frame"
-	desc = "A homemade Kalash rifle frame. Cobbled together, but as good as new. Sort of."
+	name = "Kalashnikov frame"
+	desc = "A frame for an ancient weapons system better suited for a wall mount. The receiver can take either a 6.5mm or 7.62mm barrel."
```

Comment:
I get the AK hate but this might be a bit much considering the gun is pretty serviceable balance wise.

Reply:
It's in reference to the age of the design.

Reply:
I didn't think of that, actually!